### ISSUE TITLE
[accessibility] Control should have `role="button"`

### ISSUE THREAD

**Issue by Malvoz** [consbio/Leaflet.ZoomBox#20]
The control is conveyed as <q>link</q> to screen reader users, the control should be `role="button"` ([same as Leaflet controls](https://github.com/Leaflet/Leaflet/blob/bc918d4bdc2ba189807bc207c77080fb41ecc196/src/control/Control.Zoom.js#L91)).

**Comment by brendan-ward** [comment]
Please consider submitting a PR to address this.
